src/assets/i18n/zh-hant.po

```diff
@@ -418,13 +418,13 @@ msgid "Account Name"
 msgstr "帳戶名稱"
 
 msgid ""
-"Account to be used for guest access. Default is nobody. Account is required "
+"Account to be used for guest access. ；Default is nobody. Account is required "
```

Comment:
You appear to have changed one of the string keys, could you change this one back so it does not break this key?